<?xml version="1.0"?>
<launch>

	<!-- vicon_xb node -->
	<node respawn="false" pkg="vicon_xb" type="vicon_xb_node" name="vicon_xb_node0" output="log" launch-prefix="taskset -c 2">
		<param name="viconSerialPort" type="string" value="/dev/ttyUSB0" />
		<param name="viconNodeRate" type="int" value="40"/>
		<param name="viconCommTimeout" type="int" value="250"/>

		<param name="logEnable" type="bool" value="false"/>
		<param name="blocked" type="bool" value="false"/>
		<param name="publishEnable" type="bool" value="true"/>
	</node>
	<node respawn="false" pkg="vicon_xb" type="tf_broadcaster" name="tf_broadcaster" output="log">
		<param name="pose_topic" type="string" value="/vicon_xb_node0/mocap/pose"/>
		<param name="parent_frame" type="string" value="vicon_origin"/>
		<param name="child_frame" type="string" value="uav_base"/>
	</node>
	<node pkg="tf" type="static_transform_publisher" name="vicon_local_broadcaster" 
		args="0 0 0 1.5708 0 0 vicon_origin local_origin 500" />

</launch>
